.github/workflows/demo-integrations.yml

```diff
@@ -72,23 +72,20 @@ jobs:
           curl -X GET -I -f "http://localhost:3333"
 
       - name: Run screenshot tests on demo/localhost
-        run: |
-          npm run concurrently -- \
-                             "npm run cy2:run:localhost" \
-                             "npm run cy2:run:localhost"
+        run: npm run concurrently "npm run cy2:run:localhost" "npm run cy2:run:localhost"
```

Comment:
Because e2e tests in principle don't work now in main branch

but you are right, I also made a separate PR
https://github.com/Tinkoff/taiga-ui/pull/1361/files